api/mapping/models.py

```diff
@@ -535,22 +528,40 @@ class DataDictionary(BaseModel):
 
     def __str__(self):
         return str(self.id)
-    
+
 
 class NLPModel(models.Model):
     """
     A temporary model to hold the results from NLP string searches
     Created for Sprint 14
     """
+
     user_string = models.TextField(
         max_length=1024,
     )
-    
+
     json_response = models.TextField(
         max_length=4096,
         blank=True,
         null=True,
     )
-    
+
+    def __str__(self):
+        return str(self.id)
+
+
+class ScanReportConceptID(BaseModel):
+    concept_name = models.CharField(max_length=128)
```

Comment:
With this, and many other in the models.py, could you break break the model attributes in multiple lines?

Reply:
@py5gol I can do, yes. But my implementation of PEP8 only automatically breaks onto multiple lines when the line length gets to a certain length (not sure about the number of characters though)

Reply:
I think it also splits onto multiple lines if number of attributes >2